sdk/src/state_machine/core/run.rs

```diff
@@ -114,7 +114,7 @@ impl<DaLayer: DaLayerTrait, App: StateTransitionFunction> Rollup<DaLayer, App> {
                         match self.app.apply_batch(batch, tx.sender().as_ref(), None) {
                             // TODO: handle events
                             Ok(_events) => {}
-                            Err(slashing) => current_sequencers.process_update(slashing),
+                            Err(_slashing) => todo!(), //current_sequencers.process_update(slashing),
```

Comment:
@preston-evans98 are we going to maintain the `run.rs`? My refactoring broke the logic here but I remember we said in the past that `run.rs` is not up to date anymore.

Reply:
Now that we have the demo, I think it's fine to deprecated `run.rs`

Reply:
Ok